update change miner owner address and the Addresses section of the markets config.toml (#1037)
filecoin-project/filecoin-docs: docs/mine/lotus/split-markets-miners.md

```shell
lotus-miner actor set-addrs <NEW_MULTIADDR>
```

> 9. update the Addresses section of the markets config.toml,Update this section as same  as  mining/sealing/proving node

Reviewer: ```suggestion

9. Update the `Addresses` section of the markets `config.toml` to be the same as same the mining/sealing/proving node.
```

- Add a space after each code block.
- Capitalize the start of sentences.
- Surround variable names like `Addresses` in back ticks (`).
- Surround file names like `config.toml` in back ticks (`).
- End sentences with a period/full-stop (.).

> > [Addresses]  
>   \#PreCommitControl = ["f00XX1"]  
>   \# CommitControl = ["f00XX2"]  
>     TerminateControl = []  
>   \#  DealPublishControl = ["f00XX3"] 
>     DisableOwnerFallback = false  
>     DisableWorkerFallback = false 

Reviewer: ````suggestion

```toml
[Addresses]  
  \#PreCommitControl = ["f00XX1"]  
  \# CommitControl = ["f00XX2"]  
    TerminateControl = []  
  \#  DealPublishControl = ["f00XX3"] 
    DisableOwnerFallback = false  
    DisableWorkerFallback = false 
```
````

Code examples should be in code blocks.

> Normally, if you initialize the markets node from the config.toml used in the mining/sealing/proving node, the settings in this section will be the same as in the mining/sealing/proving node, but if you are already running a subsystem and have changed the configuration in this area midway through the mining/sealing/ proving node, please make the same changes in the markets node, otherwise the DealPublishControl configuration will not take effect and DealPublish costs will be deducted from the worker wallet.see [this issue](https://github.com/filecoin-project/lotus/issues/7168)

Reviewer: ```suggestion
Typically, if you initialize the markets node from the `config.toml` used in the mining/sealing/proving node, the settings in this section will be the same as in the mining/sealing/proving node. If you are already running a subsystem and have changed the configuration in this area, copy the changes to the markets node. If you do not copy the changes, the `DealPublishControl` configuration will not affect, and `DealPublish` costs will be deducted from the worker wallet.
```
